test/NIST-800-53/IAMRules.test.ts

```diff
@@ -0,0 +1,411 @@
+/*
```

Comment:
Propose the file name be changed to 

`NIST-800-53-IAM.test.ts` to match the naming scheme of other files